src/drake/shell.clj

```diff
@@ -103,7 +103,7 @@
         proc (.exec (Runtime/getRuntime)
                     cmd-for-exec
                     ^"[Ljava.lang.String;" env
-                    (fs/file fs/*cwd*))]
+                    (fs/file (or exec-dir fs/*cwd*)))]
```

Comment:
Oops read output wrong, found the problem. Tests incoming soon to wrap up this PR.